src/routes/adminRouter/AdminRouter.tsx

```diff
@@ -3,14 +3,18 @@ import { Route, Routes } from 'react-router-dom';
 import AdminPage from '@pages/admin/AdminPage';
 import Dashboard from '@pages/manager/dashboard/Dashboard';
 import { Notices } from '@pages/manager/inquiry/notices/Notices';
+import { AdminFaq } from '@pages/manager/inquiry/faq/Faq';
+import { Management } from '@pages/product/managment/Management';
 
 // 관리자 페이지 모듈화
 export const AdminRouter = () => {
     return (
         <Routes>
             <Route path="main" element={<AdminPage />} />
             <Route path="dashboard" element={<Dashboard />} />
+            <Route path="faq" element={<AdminFaq />} />
             <Route path="notices" element={<Notices />} />
+            <Route path="prodmgt" element={<Management />} />
         </Routes>
```

Comment:
adminRouter를 만드실거면 공용으로 사용되는 주소이기 때문에 주소를 한번에 바꿔야 하는 상황이 발생하는 상황에서는 일일히 바꿔주는 일보다는 util이라는 공용적인 변수로 만들어서 처리 해주는 편이 더욱 간편할 거 같습니다. 
```suggestion
        </Routes>
```